### PR TITLE
Update issue template with links to doc tutorials

### PR DESCRIPTION
**Pull request by skyace65**
I've seen too many issues opened by people who know how to fix the issue. I think updating the issue template is a good way to give these tutorials visibility.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -11,3 +11,9 @@ assignees: ''
 **Issue description:**
 
 **URL to the documentation page:**
+
+If you know how to fix the issue you are reporting please
+consider opening a pull request. We provide a tutorial on
+using git [here](https://docs.godotengine.org/en/stable/community/contributing/pr_workflow.html)
```
Comment (pycbouh) [inline .github/ISSUE_TEMPLATE/bug_report.md:17]: Missing a comma?

**Comment by skyace65** [comment]
Fixed.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -11,3 +11,9 @@ assignees: ''
 **Issue description:**
 
 **URL to the documentation page:**
+
+If you know how to fix the issue you are reporting please
+consider opening a pull request. We provide a tutorial on
+using git [here](https://docs.godotengine.org/en/stable/community/contributing/pr_workflow.html),
+writing documentation [here](https://docs.godotengine.org/en/stable/community/contributing/docs_writing_guidelines.html)
+and contributing to the class reference [here](https://docs.godotengine.org/en/stable/community/contributing/updating_the_class_reference.html).
```
Comment (mhilbrunner) [inline .github/ISSUE_TEMPLATE/bug_report.md:19]:
```suggestion
If you know how to fix the issue you are reporting please
consider opening a pull request. We provide a tutorial on
using git here: https://docs.godotengine.org/en/stable/community/contributing/pr_workflow.html,
writing documentation at https://docs.godotengine.org/en/stable/community/contributing/docs_writing_guidelines.html
and contributing to the class reference here: https://docs.godotengine.org/en/stable/community/contributing/updating_the_class_reference.html
```

**Comment by mhilbrunner** [comment]
Changed those markdown links to plaintext, as it may be less confusing in the default (edit) view when creating a new issue. Otherwise looks good to merge, thank you!

**Pull request merged**